zuora.gemspec

```diff
@@ -1,36 +1,34 @@
-# -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "zuora/version"
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+version = File.read(File.expand_path('ZUORA_VERSION', __dir__)).strip
 
 Gem::Specification.new do |s|
-  s.name        = "zuora"
-  s.version     = Zuora::Version.to_s
-  s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Josh Martin"]
-  s.email       = ["josh.martin@wildfireapp.com"]
-  s.homepage    = "http://github.com/wildfireapp/zuora"
-  s.summary     = %q{Zuora - ActiveModel backed client for the Zuora API}
-  s.description = %q{Zuora - Easily integrate the Zuora SOAP API using ruby objects.}
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
-  s.require_paths = ["lib"]
-  s.extra_rdoc_files = [ "README.md" ]
+  s.name        = 'zuora'
+  s.version     = version
+  s.authors     = ['Josh Martin']
+  s.email       = ['josh.martin@wildfireapp.com']
+  s.summary     = 'Zuora - ActiveModel backed client for the Zuora API'
+  s.description = 'Zuora - Easily integrate the Zuora SOAP API using ruby objects.'
+  s.homepage    = 'https://github.com/zuorasc/zuora'
 
-  s.add_runtime_dependency('savon', ["0.9.9"])
-  s.add_runtime_dependency('activesupport', [">= 3.0.0", "< 5.0.0"])
-  s.add_runtime_dependency('activemodel', [">= 3.0.0", "< 5.0.0"])
-  s.add_runtime_dependency('httpi', ["0.9.7"])
-  s.add_runtime_dependency('libxml4r', ['~> 0.2.6'])
+  s.files       = Dir.glob('{bin,lib}/**/**/**')
+  s.bindir = 'exe'
+  s.executables = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  s.require_paths = ['lib']
+  s.platform    = Gem::Platform::RUBY
+  s.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
+  s.extra_rdoc_files = ['README.md']
 
-  s.add_development_dependency('rake', ["~> 0.8.7"])
-  s.add_development_dependency('guard-rspec', ["~> 0.6.0"])
-  s.add_development_dependency('artifice', ["~> 0.6.0"])
-  s.add_development_dependency('yard', ["~> 0.7.5"])
-  s.add_development_dependency('rspec', ["~> 2.8.0"])
-  s.add_development_dependency('redcarpet', ["~> 2.1.0"])
-  s.add_development_dependency('factory_girl', ["~> 2.6.4"])
-  s.add_development_dependency('appraisal', ["~> 0.4.1"])
-  s.add_development_dependency('sqlite3', ["~> 1.3.0"])
-  s.add_development_dependency('simplecov', ["~> 0.6.4"])
+  s.add_dependency 'activemodel', '< 5.0.0'
+  s.add_dependency 'activesupport', '< 5.0.0'
+  s.add_dependency 'akami', '1.3.2'
+  s.add_dependency 'ffi', '1.16.3'
+  s.add_dependency 'httpi', '~> 1.0'
+  s.add_dependency 'libxml4r', '~> 0.2.6'
+  s.add_dependency 'nokogiri', '~> 1.15.6'
+  s.add_dependency 'ruby3-backward-compatibility'
+  s.add_dependency 'savon', '~> 0.9.12'
+  s.add_dependency 'wasabi', '2.3.0'
```

Comment:
We seem to have doubled the non-development dependencies from 5 to 10. Are all these changes necessary?

`ruby3-backward-compatibility` definitely isn't needed here.

Reply:
The two Gems below are the ones I introduced to resolve a couple of dependency errors encountered:
```
akami
ffi
```
The ones below already exists in the Gemfile from `master` branch, I brought them into this file in order to have them arranged in alphabetical order.
```
savon
httpi
wasabi
```

Reply:
Are these all runtime dependencies? All the development dependencies seem to have disappeared from here and are in the Gemfile instead?

You still need to remove `ruby3-backwards-compatibility`.

Also we don't want to lock gems to specific versions in the gemspec. We should be as permissive as possible with gem version specs.

Reply:
`savon, httpi and wasabi` appears to be runtime dependencies, while `akami and ffi` are development dependencies. FIxing in the the PR update.

Reply:
`wasabi` should just go in the Gemfile as it's a dependency of `savon`.